### PR TITLE
Bufix: Fixes zos_copy and zos_fetch deprecation msg for using _play_context.verbosity

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
This addresses issue #779 where a deprecation warning would appear when zos_copy or zos_fetch ran. Of the two modules, only the `zos_copy` module  could be automated with a test because its plugin is invoked immediately where as with `zos_fetch` its invoked last and that can not work for testing display.verbosity.

In addition to this, the test case for #616 was addressed here because the original test case could not run properly on local work stations and the pipeline. So a new test case was written to run on both.

The pytest-ansible fixture was updated to support the `zos_copy` adhoc test case.

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by ddimatos** [comment]
@fernandofloresg - I requested you since you have similar work to do for the archive modules.
@AndreMarcel99 - I requested you because of the issue #616 , you had worked on it.

**Comment by ddimatos** [comment]
Pipeline job `job/zosAnsibleAlternate2/51/console`
`======================= 541 passed in 8499.99s (2:21:39) =======================`
image

**Review by fernandofloresg — CHANGES_REQUESTED** [review]

File: tests/functional/modules/test_zos_copy_func.py [hunk 1]
```diff
@@ -470,45 +471,99 @@ def test_copy_dir_to_existing_uss_dir_not_forced(ansible_zos_module):
...
+
+        for result in str_three_od_dst.contacted.values():
+            assert result.get("stdout") == str_three_big_endian_hex
 
-        assert text_inner == text_inner_file
-        assert text_outer == text_outer_file
     finally:
-        hosts.all.file(path=src_path, state="absent")
-        hosts.all.file(path=dest_path, state="absent")
+       hosts.all.file(path=path, state="absent")
```
[the remark this answers is not in the document]
Comment (ddimatos) [reply tests/functional/modules/test_zos_copy_func.py]: Its corrected now.

**Review by AndreMarcel99 — CHANGES_REQUESTED** [review]

File: changelogs/fragments/791-zos_data_set-update-vsam-copy.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+bugfixes:
+- zos_copy - Test case for recursive encoding directories reported a
+  UTF-8 failure. This change corrects and ensures the proper test coverage
```
[the remark this answers is not in the document]
Comment (ddimatos) [reply changelogs/fragments/791-zos_data_set-update-vsam-copy.yml:3]: Ill update it but:
corrects - fixes the bug
ensures - is saying it adds a test case. 

If no test case was added , corrects only would have been used.

File: tests/functional/modules/test_zos_copy_func.py [hunk 3]
```diff
@@ -2868,4 +2923,40 @@ def test_copy_uss_file_to_existing_sequential_data_set_twice_with_tmphlq_option(
...
+
+        # This is an adhoc command, because there was no
+        cmd = "ansible all -i " + str(node) + ", -u " + user + " -m ibm.ibm_zos_core.zos_copy -a \"src=" + options["src"] + " dest=" + options["dest"] + " is_remote=" + str(
+            options["is_remote"]) + " encoding={{enc}} \" -e '{\"enc\":{\"from\": \"ISO8859-1\", \"to\": \"IBM-1047\"}}' -e \"ansible_python_interpreter=" + python_path + "\" " + options["verbosity"] + ""
+
+        result = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE).stdout
+        output = result.read().decode()
+
+        if options["verbosity_level"] != 0:
+            assert ("play context verbosity: "+ str(options["verbosity_level"])+"" in output)
```
[the remark this answers is not in the document]
Comment (ddimatos) [reply tests/functional/modules/test_zos_copy_func.py:2957]: It seems as if you want if `options["verbosity_level"]` set to a variable, that is fine but note that our pipe lines have other priorities now and this change should probably require a new pipeline run. Not sure we can spare the compute at this time to for a net zero change. 

Let me know.

**Comment by ddimatos** [comment]
@fernandofloresg - changes made, I think you can resolve yours now. 
@AndreMarcel99 - one is resolved the other i had a comment to, let me know.

File: tests/functional/modules/test_zos_copy_func.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (AndreMarcel99) [reply tests/functional/modules/test_zos_copy_func.py:2957]: Ok. I'll going to add next iteration as a small change just to keep the consistency.

**Review by fernandofloresg — APPROVED** [review]

**Review by AndreMarcel99 — APPROVED** [review]

**Pull request merged**